<!doctype html><html lang="en-us">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    
<meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="generator" content="Qubt theme for Hugo" />



<title>Prajwal Bharadwaj</title>
<meta name="title" content="Prajwal Bharadwaj" />



  <meta name="author" content="Hi! I am Prajwal Bharadwaj" />




  <meta name="description" content="Demo site build with Qubt &amp; Hugo" />









<link type="text/css" rel="stylesheet" href="/css/main.bundle.min.c23b5d74caf13c24f43a3c971099a56d9c922bfd7b3743ab3d259b5c1e96fbf4.css" integrity="sha256-wjtddMrxPCT0OjyXEJmlbZySK/17N0OrPSWbXB6W&#43;/Q=" />









<script defer src="/js/main.bundle.min.5a00293bacd8510f50add105212cea76d541a955f6693035ef3128b638a21619.js" integrity="sha256-WgApO6zYUQ9QrdEFISzqdtVBqVX2aTA17zEotjiiFhk="></script>






<meta property="og:url" content="//localhost:1313/cv/">
  <meta property="og:site_name" content="Prajwal Bharadwaj">
  <meta property="og:title" content="Work Experience">
  <meta property="og:description" content="PhD with Dr. James Hall Plasmid-Host Dynamics in the context of Antibiotic Resistance October 2023 - ongoing
Studying persistence of antimicrobial resistance as a function of plasmid persistence in the host Understanding the interaction dynamics of the plasmid and the host genome Research Project with Dr. Samay Pande Research Project on evolution of antibiotic resistance August 2022 - June 2023
Understanding the influence of multiple selection-constrains on the de-novo evolution of antibiotic resistance Studying evolution of antibiotic resistance in prey bacteria in the presence of a predator and evolution of escape strategies by the prey. The model system used was of M xanthus (predator) and E coli, P fluorescence and S enterica (prey). Experimental coevolution of M xanthus and three prey bacteria - Microbial culture techniques Analysis of predatory efficiency of and competition between clones of M xanthus from a population mixing evolution experiment Standardisation of NGS based Frequency Analysis of M xanthus and E coli clones Masters Thesis with Dr. Frederic Bertels Studying the dynamics of REPIN movement in bacterial genomes March 2021 - June 2022">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="og:image" content="//localhost:1313/thumbnail.png">


  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:image" content="//localhost:1313/thumbnail.png">
  <meta name="twitter:title" content="Work Experience">
  <meta name="twitter:description" content="PhD with Dr. James Hall Plasmid-Host Dynamics in the context of Antibiotic Resistance October 2023 - ongoing
Studying persistence of antimicrobial resistance as a function of plasmid persistence in the host Understanding the interaction dynamics of the plasmid and the host genome Research Project with Dr. Samay Pande Research Project on evolution of antibiotic resistance August 2022 - June 2023
Understanding the influence of multiple selection-constrains on the de-novo evolution of antibiotic resistance Studying evolution of antibiotic resistance in prey bacteria in the presence of a predator and evolution of escape strategies by the prey. The model system used was of M xanthus (predator) and E coli, P fluorescence and S enterica (prey). Experimental coevolution of M xanthus and three prey bacteria - Microbial culture techniques Analysis of predatory efficiency of and competition between clones of M xanthus from a population mixing evolution experiment Standardisation of NGS based Frequency Analysis of M xanthus and E coli clones Masters Thesis with Dr. Frederic Bertels Studying the dynamics of REPIN movement in bacterial genomes March 2021 - June 2022">


<script>
  
  if (localStorage.getItem("color-theme") === "dark" || (!("color-theme" in localStorage) && window.matchMedia("(prefers-color-scheme: dark)").matches)) {
    document.documentElement.classList.add("dark");
  } else {
    document.documentElement.classList.remove("dark");
  }
</script>

  </head>
  <body class="flex h-screen flex-col justify-between bg-neutral-100 dark:bg-neutral-800">
    <div>
      <header class="sticky top-0 z-10 bg-neutral-100 dark:bg-neutral-800">
        



<section class="mx-auto flex max-w-screen-xl items-center justify-between p-4">
  <a href="//localhost:1313/" class="flex items-center space-x-3">
    
    <span class="self-center whitespace-nowrap text-2xl font-semibold text-slate-700 dark:text-slate-400">
      Prajwal Bharadwaj
    </span>
  </a>
  <div class="flex flex-row space-x-8">
    <nav class="hidden space-x-8 text-xl md:block" aria-label="main">
      
        <a href="/science" class="px-3 py-2 text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
          Science
        </a>
      
        <a href="/scicom" class="px-3 py-2 text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
          SciCom
        </a>
      
        <a href="/blog/" class="px-3 py-2 text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
          Blog
        </a>
      
        <a href="/cv/" class="px-3 py-2 text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
          CV
        </a>
      
        <a href="/about/" class="px-3 py-2 text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
          About
        </a>
      
    </nav>
        <path d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"></path>
      </svg>
        <path
          d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
          fill-rule="evenodd"
          clip-rule="evenodd"></path>
      </svg>
    </button>
    <button id="hamburger-button" class="relative h-8 w-8 cursor-pointer text-3xl md:hidden" aria-label="hamburger-button">
      <div
        class="absolute top-4 -mt-0.5 h-[3px] w-8 rounded bg-slate-700 transition-all duration-500 before:absolute before:h-[3px] before:w-8 before:-translate-x-4 before:-translate-y-2.5 before:rounded before:bg-slate-700 before:transition-all before:duration-500 before:content-[''] after:absolute after:h-[3px] after:w-8 after:-translate-x-4 after:translate-y-2.5 after:rounded after:bg-slate-700 after:transition-all after:duration-500 after:content-[''] dark:bg-slate-400 before:dark:bg-slate-400 after:dark:bg-slate-400"></div>
    </button>
  </div>
</section>
<section id="mobile-menu" class="absolute hidden w-full origin-top animate-open-menu flex-col justify-center bg-neutral-100 text-4xl dark:bg-neutral-800">
  <nav class="flex min-h-screen flex-col items-center py-8" aria-label="mobile">
    
      <a href="/science" class="px-3 py-2 text-center text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
        Science
      </a>
    
      <a href="/scicom" class="px-3 py-2 text-center text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
        SciCom
      </a>
    
      <a href="/blog/" class="px-3 py-2 text-center text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
        Blog
      </a>
    
      <a href="/cv/" class="px-3 py-2 text-center text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
        CV
      </a>
    
      <a href="/about/" class="px-3 py-2 text-center text-slate-700 hover:text-indigo-500 dark:text-slate-400 md:p-0">
        About
      </a>
    
  </nav>
</section>

      </header>
      <main>
        
  
  
  
  
  


  <div class="justify-left mx-auto mt-8 flex max-w-screen-md px-4">
    <article>
      <h1 class="text-4xl font-extrabold text-slate-700 dark:text-slate-200">
        Work Experience
      </h1>
      <h2 class="mt-4 text-2xl text-slate-500 dark:text-slate-400">
        
      </h2>

      

      


      <span class="prose prose-slate break-words text-lg text-slate-700 dark:prose-invert prose-pre:max-w-[90vw] dark:text-slate-200 md:prose-pre:max-w-screen-md">
        <h2 id="phd-with-dr-james-hall">PhD with Dr. James Hall</h2>
<p><strong>Plasmid-Host Dynamics in the context of Antibiotic Resistance</strong>
<em>October 2023 - ongoing</em></p>
<ul class='container'>
    <li> Studying persistence of antimicrobial resistance as a function of plasmid persistence in the host </li>
    <li> Understanding the interaction dynamics of the plasmid and the host genome </li>
</ul>
<hr>
<h2 id="research-project-with-dr-samay-pande">Research Project with Dr. Samay Pande</h2>
<p><strong>Research Project on evolution of antibiotic resistance</strong>
<em>August 2022 - June 2023</em></p>
<ul class='container' style='display list-item;'>
<li> Understanding the influence of multiple selection-constrains on the de-novo evolution of antibiotic resistance</li>
<li> Studying evolution of antibiotic resistance in prey bacteria in the presence of a predator and evolution of escape strategies by the prey. The model system used was of M xanthus (predator) and E coli, P fluorescence and S enterica (prey).</li>
<li> Experimental coevolution of M xanthus and three prey bacteria - Microbial culture techniques</li>
<li> Analysis of predatory efficiency of and competition between clones of M xanthus from a population mixing evolution experiment</li>
<li> Standardisation of NGS based Frequency Analysis of M xanthus and E coli clones</li>
</ul>
<hr>
<h2 id="masters-thesis-with-dr-frederic-bertels">Masters Thesis with Dr. Frederic Bertels</h2>
<p><strong>Studying the dynamics of REPIN movement in bacterial genomes</strong>
<em>March 2021 - June 2022</em></p>
<ul class='container'>
<li>Understanding movement patterns and duplication rate of REPIN Transposable Elements in Pseudomonas genome sequences</li>
<li>Creating and testing of REPIN Clusterer - a Bioinformatics pipelines in Python Programming Language </li>
<li>Developed innovative computational method of clustering REPIN Sequences together based on orthology </li>
<li>Established events of duplication of REPIN Sequences based on the Clusters and calculated their rate of duplication. </li>
<li>Critical analysis of potential theories on movement of REPIN within host genome </li>
<li>Automated BLAST searches, isolated genes, compared genes, built, compared and analysed phylogenetic trees, curated computer code and datasets</li>
</ul>
<hr>
<h2 id="igem-2020-project">iGEM 2020 Project</h2>
<p><strong>Beta-cells in Gut Producing Insulin (BIG-PIE)</strong>
<em>February 2020 - October 2020</em></p>
<ul class='container' style='24px'>
<li>A new method of providing insulin to patients with diabetes</li>
<li>Helped Brainstorm project ideas during the nascent stage</li>
<li>Helped with building the Project Website</li>
<li>Contributed to Outreach and Science Communication by conducting seminars for School students</li>
<li>Gold Medal Winning Project at The Giant Jamboree, iGEM 2020</li>
</ul>
<hr>
<h2 id="igem-2019-project">iGEM 2019 Project</h2>
<p><strong>Ecoli Learns Suboptimal Acclimatisation (ELSA)</strong>
<em>February 2019 - October 2019</em></p>
<ul class='container'>
<li> Synthetically modifying Ecoli to grow optimally at 4 degrees Celsius</li>
<li> Helped manage the Wet Lab experiments by performing Growth Curve experiments to analyse the effect of our Gene of Interest</li>
<li> Helped the Outreach and Integrated Human Practices team by talking to people in the industry regarding our product.</li>
<li> Silver Medal Winning Project at The Giant Jamboree, iGEM 2019</li>
</ul>

      </span>
    </article>
  </div>

      </main>
    </div>
  </body>
</html>
